OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -897,27 +900,34 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Millard2012AccelerationMuscle, Muscle);
                                           const AccelerationMuscleInfo& ami,
                                           std::string& caller) const;
 
-    //=====================================================================
-    // Private Utility Class Members
-    //      -Computes activation dynamics and fiber kinematics
-    //=====================================================================
+    // Status flag returned by initMuscleState().
+    enum ResultOfInitMuscleState {
+        Success_Converged,
+        Warning_FiberAtLowerBound,
+        Failure_MaxIterationsReached
+    };
+
```

Comment:
> If this enum can be generalized to be applicable to other all muscles, consider moving the enum out of the class and making it an `enum class` (It offers lot more type safety and regular enum too).
Moving it out of the class may have implications toward bindings.

The enum is not a common feature of all muscles since the internal `initMuscleState()` and `estimateMuscleFiberState()` methods will not necessarily appear in every muscle model. In fact, `Millard2012AccelerationMuscle::initMuscleState()` and `Thelen2003Muscle::initMuscleState()` are both private methods that are called in exactly one place, so there does not seem to be any reason not to embed them in their calling methods (`computeInitialFiberEquilibrium()` in both cases). However, I think it is beyond the scope of this PR to do further refactoring.